core/templates/dev/head/pages/practice-session-page/practice-session-page.directive.html

```diff
@@ -0,0 +1,33 @@
+<background-banner></background-banner>
+<div class="oppia-practice_session-container" style="position: relative;">
+  <div class="oppia-practice-session">
+    Practice Session
+  </div>
+  <question-player ng-if="$ctrl.questionPlayerConfig.skillList &&
+                          $ctrl.questionPlayerConfig.skillList.length > 0"
+                   player-config="$ctrl.questionPlayerConfig">
+  </question-player>
+</div>
+<style>
+  .oppia-practice_session-container .oppia-practice-session {
+    color: #01645c;
+    font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+    font-size: 3em;
+    margin-top: 0px;
+    margin-bottom: 25px;
+    padding-top: 66.5px;
+    text-align: center;
+  }
+  .oppia-practice_session-container {
+    color: black;
+    font-family: 'Roboto', Arial, sans-serif;
+    margin: 0 auto;
+    text-decoration: none;
+  }
+
+  @media(max-width: 925px) {
+    .oppia-practice_session-containerr {
```

Comment:
Is this a typo? It should be a single r at the end right?

Reply:
I don't know, I'll have to check since this was in the original file. I did not modify this. Most probably this is a typo.